lib/tasks/npm-task.js

```diff
@@ -124,37 +158,50 @@ class NpmTask extends Task {
    * @method findPackageManager
    * @return {Promise}
    */
-  async findPackageManager() {
-    if (this.useYarn === true) {
+  async findPackageManager({ useYarn } = {}) {
```

Comment:
I'd like for this to be modified (e.g. `packageManager` string instead of a boolean for `useYarn`).

Reply:
I was gonna comment on the changes to `new` and `addon` command, but looks like you didn't need to make any modifications.

I'd like to make `--package-manager` option, so you could do `--package-manager=yarn` or `--package-manager=pnpm`. Then `--yarn` would be an alias for `--package-manager=yarn` (and we could add `--pnpm` as an alias of `--package-manager=pnpm`).

Reply:
yeah, I agree. I wanted to keep the API of the class the same for now though.

Reply:
I've added support for `--package-manager` now.